lib/resources/pip.rb

```diff
@@ -62,19 +62,56 @@ def to_s
 
     private
 
+    def cmd
+      @__cmd ||= inspec.command("#{@pip_cmd} show #{@package_name}")
+    end
+
+    def cmd_successful?
+      result = cmd
```

Comment:
Seems like every instance of `result` in this method could just be `cmd`? (And would actually read better that way?)